src/components/pages/landingpage/customers.tsx

```diff
@@ -66,23 +66,63 @@ const CustomerLogo = ({
   );
 };
 
+const AriaCustomerLogo = () => {
+  return (
+    <ul
+      style={{
+        position: 'absolute',
+        width: '1px',
+        height: '1px',
+        padding: '0',
+        margin: '-1px',
+        overflow: 'hidden',
+        clip: 'rect(0, 0, 0, 0)',
+        whiteSpace: 'nowrap',
+        border: '0',
+        listStyleType: 'none',
+      }}
+      aria-label={'Our Customers:'}
+    >
+      <li>Allianz</li>
+      <li>Rosenbauer</li>
+      <li>Media Markt</li>
+      <li>Saturn</li>
+      <li>Green City</li>
+      <li>FC Bayern</li>
+      <li>Zeiss</li>
+      <li>A.D.A.C</li>
```

Comment:
The dots introduce some pause when read by the screen readers?

Reply:
yes. without the dots VoiceOver tried to read it as one word instead of an acronym.